src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color='" + color + '\''
+                + ", number=" + number
+                + '}';
+    }
+}

```

Comment:
Consider defining a package-private or public constructor for the Ball class to ensure that it can be instantiated from other classes in the same package or from anywhere, respectively. By default, the constructor is package-private, which may be fine if instantiation is only needed within the same package, but if you plan to use Ball instances elsewhere, a public constructor is needed.